src/app/tests/suites/certification/Test_TC_OO_1_1.yaml

```diff
@@ -38,38 +38,75 @@ tests:
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
-          value: 5
+          value: 6
           constraints:
               type: int16u
 
-    - label: "Step 3a: TH reads from the DUT the FeatureMap attribute."
+    - label:
+          "Step 3a: Given OO.S.F00(LT) ensure featuremap has the correct bit set"
       command: "readAttribute"
       attribute: "FeatureMap"
-      PICS: ( !OO.S.F00 && !OO.S.F01 )
+      PICS: ( OO.S.F00 && !OO.S.F02 )
       response:
-          value: 0
           constraints:
               type: bitmap32
-
+              hasMasksSet: [0x1]
     - label:
           "Step 3b: Given OO.S.F00(LT) ensure featuremap has the correct bit set"
       command: "readAttribute"
       attribute: "FeatureMap"
-      PICS: OO.S.F00
+      PICS: ( !( OO.S.F00 && !OO.S.F02 ) )
       response:
           constraints:
               type: bitmap32
-              hasMasksSet: [0x1]
+              hasMasksClear: [0x1]
 
     - label:
           "Step 3c: Given OO.S.F01(DF) ensure featuremap has the correct bit set"
       command: "readAttribute"
       attribute: "FeatureMap"
-      PICS: OO.S.F01
+      PICS: ( OO.S.F01 && !OO.S.F02 )
       response:
           constraints:
               type: bitmap32
               hasMasksSet: [0x2]
+    - label:
+          "Step 3d: Given OO.S.F01(DF) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: ( !( OO.S.F01 && !OO.S.F02 ) )
```

Comment:
Same, why not split into 2 for easier understanding

Reply:
please check other responses above.